pkg/tools/etcd_helper_test.go

```diff
@@ -685,7 +687,7 @@ func TestGuaranteedUpdate_CreateCollision(t *testing.T) {
 
 func TestGetEtcdVersion_ValidVersion(t *testing.T) {
 	testServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		fmt.Fprint(w, "etcd 2.0.9")
+		fmt.Fprint(w, etcdVersion209)
```

Comment:
This is really only intended to be a sample result of what the /version endpoint can return. We don't parse the version, so I think just this one string is enough.